src/modules/Providers/views/migrate/components/PlansCreateForm.tsx

```diff
@@ -44,7 +45,7 @@ import { MappingListHeader } from './MappingListHeader';
 import { StateAlerts } from './StateAlerts';
 
 const buildNetworkMessages = (
-  t: (key: string) => string,
+  t: TFunction,
```

Comment:
I think it is easier instead of adding a type, simply import t directly here and omit it from comment